### ISSUE TITLE
Parsing of replay ends with error: CDOTAPlayer does not have field: m_iPlayerID

### ISSUE THREAD

**Issue by vlejd** [timkurvers/redota#63]
New replays end with error 
Tried for 6456813156 and 6455478307
Old replays work fine.

Error log:
```
Error: entity CDOTAPlayer does not have field: m_iPlayerID
    at get@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:1:57370
    at processPlayer@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:551:5230
    at onEntities@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:551:2438
    at 7187/i.prototype.emit@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:148540
    at onCSVCMsg_PacketEntities@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:1:52802
    at 7187/i.prototype.emit@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:148540
    at onCDemoPacket@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:1:48620
    at 7187/i.prototype.emit@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:148540
    at seek@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:1:48130
    at step@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:1:47646
    at start@https://timkurvers.github.io/redota/redota-parser+viewer-f20ee975.js:1:47624
    at ve@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:171821
    at o@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:171362
    at 2129/m/<@https://timkurvers.github.io/redota/redota-main-a667d387.js?82f7cdcab7c725abab2c:213:964
    at Ns@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:317045
    at 53/t.unstable_runWithPriority@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:360235
    at Vo@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:257583
    at Ps@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:316508
    at 4448/As/<@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:316419
    at B@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:359296
    at 53/w.port1.onmessage@https://timkurvers.github.io/redota/redota-vendor-6c7c4893.js?82f7cdcab7c725abab2c:2:358019
    at 

```

**Comment by timkurvers** [comment]
Thanks for the report!

It seems Valve has introduced `m_nPlayerID` on CDOTAPlayer, but it does not follow the same 0-9 values that `m_iPlayerID` previously had, which throws off color indexing (among other things).

Hope to have a fix ready this week. May also tie into #46, which is related to player associations.

**Comment by timkurvers** [comment]
Addressed in 7a4ab8b663082de1d090cd9a2a62587d024411f5 and should work for both old replays as well as anything on patch 7.31.

New build is available on website.

**Comment by vlejd** [comment]
Awesome!
